sageodata-apispec.yml

```diff
@@ -727,6 +727,84 @@ paths:
           description: Success
       operationId: GetMonitoringNetworks
       summary: Get available Monitoring Networks
+  /api/v1.0/elevationmeasurements/:
+    summary: Endpoint to interact with elevation measurements.
+    description: ""
+    get:
+      tags:
+      - obswell
+      parameters:
+      - name: modifiedSince
+        description: If specified, only elevation measurements that have been modified
+          more recently than the supplied value will be returned.
+        schema:
+          format: date-time
+          type: string
+        in: query
+      - name: groupCode
+        description: If specified, only measurements recorded for drillholes with
+          the matching group code will be returned.
+        schema:
+          type: string
+        in: query
+        required: false
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/elevationmeasurement-page'
+          description: Successful operation
+      operationId: GetElevationMeasurements
+      summary: List all elevation measurements
+      description: ""
+    parameters:
+    - name: id
+      description: DrillholeId specifying which obswell to retrieve measurements for.
+      schema:
+        format: int64
+        type: integer
+      in: path
+      required: true
+  /api/v1.0/waterlevelmeasurements/:
+    summary: Endpoint to interact with water level measurements.
+    description: ""
+    get:
+      tags:
+      - obswell
+      parameters:
+      - name: modifiedSince
+        description: If specified, only water level measurements that have been modified
+          more recently than the supplied value will be returned.
+        schema:
+          format: date-time
+          type: string
+        in: query
+      - name: groupCode
+        description: If specified, only measurements recorded for drillholes with
+          the matching group code will be returned.
+        schema:
+          type: string
+        in: query
+        required: false
+      responses:
+        "200":
```

Comment:
Same comment as for the `elevationmeasurements`

We will have to add some conditions to call this endpoint.
1. `groupCode` and `modifiedSince` are mandatory
2. `modifiedSince` cannot be greater than (say) 1 year

This is a necessary limitation to control the volume of the returned dataset and also allow better performance from the API. If these conditions are not met, a standard 40x error code can be sent back to the client.